controllers/postComment.js

```diff
@@ -0,0 +1,37 @@
+const { insertComment } = require("../models/insertComment");
+const { fetchArticleById } = require("../models/article_id");
+
+exports.postComment = (req, res, next) => {
+  const { username, body } = req.body;
+  const article_id = req.params.article_id;
+
+  if (isNaN(article_id)) {
+    const err = new Error("Bad request");
+    err.status = 400;
+    throw err;
+  }
+  if (!username || !body) {
+    const err = new Error("Bad request");
+    err.status = 400;
+    throw err;
+  }
+
+  fetchArticleById(article_id)
+    .then((article) => {
+      if (!article) {
+        const err = new Error("Not found");
```

Comment:
This logic should be handled in the model. You know if you get back an empty array that the article_id was not in the database, so you can use Promise.reject() to throw an error